.github/schema-store-issue-template.md

```diff
@@ -21,9 +20,14 @@ Extend the catalog entry found in `src/api/json/catalog.json` to include the new
     "*.flagd.yaml",
     "*.flagd.yml"
     ],
-    "url": "https://raw.githubusercontent.com/open-feature/schemas/main/json/flagd-definitions.json",
+    "url": "https://flagd.dev/schema/v0/flags.json",
     "versions": {
     "0.1.1": "https://raw.githubusercontent.com/open-feature/schemas/json/json-schema-v0.1.1/json/flagd-definitions.json",
 +   "X.X.X": "https://raw.githubusercontent.com/open-feature/schemas/json/json-schema-vX.X.X/json/flagd-definitions.json"
 }
-```
\ No newline at end of file
+```
```

Comment:
Now this file suggests reviewing the schemastore version, and updating the version hosted at flagd.dev